nasdf/compilation-tests.lisp

```diff
@@ -37,24 +42,48 @@ A sub-package has a name that starts with that of PACKAGE followed by a '/' sepa
 (defun list-subpackages (package)
   (remove-if (lambda (pkg) (not (subpackage-p pkg package))) (list-all-packages)))
 
-(defun unbound-exports (package)
-  "Report unbound exported symbols for PACKAGE and all its subpackages."
-  ;; TODO: Only SBCL is supported for now.
-  #-sbcl
-  nil
-  #+sbcl
-  (let* ((package (find-package package))
-         (report (delete nil
-                         (mapcar (lambda (package)
-                                   (let ((exports (list-unbound-exports package)))
-                                     (when exports
-                                       (list package exports))))
-                                 (cons (find-package package) (list-subpackages package))))))
-    (when report
-      (error "~a~&Found unbound exported symbols in ~a package~:p."
-             report (length report)))))
+(defun list-undocumented-exports (package)
+  (let ((result '()))
+    (do-external-symbols (s (find-package package) result)
+      (unless (or (documentation s 'variable)
+                  (documentation s 'function)
+                  (documentation s 'compiler-macro)
+                  (documentation s 'setf)
+                  (documentation s 'method-combination)
+                  (documentation s 'type)
+                  (documentation s 'structure)
+                  (and (find-package :parenscript)
+                       (gethash s (symbol-value (find-symbol "*MACRO-TOPLEVEL*" :parenscript)))))
```

Comment:
Wait, you are not looking for the docstring here, are you?

Reply:
No, I'm excluding the Parenscript symbols, because they are unlikely to have documentation.

Reply:
OK, maybe add a comment then, cause otherwise this is confusing.